dd-sdk-android/src/main/kotlin/com/datadog/android/error/internal/DatadogExceptionHandler.kt

```diff
@@ -69,6 +74,14 @@ internal class DatadogExceptionHandler(
         )
     }
 
+    private fun createCrashMessage(throwable: Throwable): String {
+        return if (throwable.message != null) {
```

Comment:
I'd test if the message is empty or blank too

Reply:
yes, makes sense. nice catch!